apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/util/ThrowableTransformer.java

```diff
@@ -58,6 +58,9 @@ private String printExceptionInfo(Throwable causeException) {
     }
 
     private boolean printStackElement(StackTraceElement[] stackTrace, AppendListener printListener) {
+        if (stackTrace.length == 0) {
+            return true;
```

Comment:
Why return `true` could void OOM? If I am reading code right, this root cause is `Exception` is parent endless loop?

Reply:
it seems cause by he has override fillInStackTrace
![image](https://user-images.githubusercontent.com/31895647/59925991-deb00580-946b-11e9-9e54-0d2afb60a8e4.png),but i don't know what's the mean of this.

Reply:
Because when SentinelRpcException happens,printListener.overMaxLength() is never be invoked,then the value of overMaxLength is always false.so it cause endless loop.
![image](https://user-images.githubusercontent.com/31895647/59926322-a9f07e00-946c-11e9-88e2-9ef2a113a7c6.png)